src/sentry/static/sentry/app/plugins/jira/components/issueActions.tsx

```diff
@@ -3,14 +3,22 @@ import React from 'react';
 import {Form, FormState} from 'app/components/forms';
 import DefaultIssueActions from 'app/plugins/components/issueActions';
 
+//taken from https://stackoverflow.com/questions/46634876/how-can-i-change-a-readonly-property-in-typescript
+type Writable<T> = {-readonly [K in keyof T]: T[K]};
```

Comment:
Should we move this somewhere else where it can be reused if needed in the future?